app/controllers/users_controller.rb

```diff
@@ -1,7 +1,66 @@
 class UsersController < ApplicationController
 
-  def list
+  def show
     @users = User.all
   end
-  alias :index :list
+  alias :index :show
+
+  def new
+  end
+
+  def create
+    @user = User.create(user_params)
+    redirect_to '/'
```

Comment:
Would be nice to put a flash message in there to show the user upon returning them to the homepage.

http://guides.rubyonrails.org/action_controller_overview.html#the-flash